_posts/2025-01-16-19.0.0-release.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: "Apache Arrow 19.0.0 Release"
+date: "2025-01-16 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 19.0.0 release. This release
+covers over 2 months of development work and includes [**202 resolved
+issues**][1] on [**330 distinct commits**][2] from [**67 distinct
+contributors**][2]. See the [Install Page](https://arrow.apache.org/install/) to
+learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 18.1.0 release, Adam Reeve and Laurent Goujon have been invited to
+become committers. Gang Wu has been invited to join the Project Management
+Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Columnar format
+
+TODO
+
+## Arrow Flight RPC notes
+
+TODO
+
+## C++ notes
+
+TODO
+
+### Acero
+
+TODO
+
+### Compute
+
+TODO
+
+### Dataset
+
+
+### Filesystems
+
+TODO
+
+### Gandiva
+
+TODO
+
+### GPU
+
+
+### IPC
+
+TODO
+
+### Parquet
+
+TODO
+
+## C# notes
+
+TODO
```

Comment:
@CurtHagenlocher @adamreeve can you help with the C# release notes?

Reply:
```suggestion
* The `PrimitiveArrayBuilder` constructor has been made public to allow writing custom builders [GH-23995](https://github.com/apache/arrow/issues/23995)
* Improved the performance of looking up schema fields by name [GH-44575](https://github.com/apache/arrow/issues/44575)
```

Reply:
Looks good, thanks @adamreeve.